examples/python/pincell_depletion/run_depletion.py

```diff
@@ -0,0 +1,161 @@
+#                   Exporting to OpenMC materials.xml file
+###############################################################################
+
+# Compute cell areas
+area = {}
+area[fuel] = np.pi * fuel_or.coefficients['R'] ** 2
+
+# Set materials volume for depletion. Set to an area for 2D simulations
+uo2.volume = area[fuel]
+
+# Instantiate a Materials collection and export to XML
+materials_file = openmc.Materials([uo2, helium, zircaloy, borated_water])
+materials_file.export_to_xml()
+
+###############################################################################
+#                   Exporting to OpenMC settings.xml file
+###############################################################################
+
+# Instantiate a Settings object, set all runtime parameters, and export to XML
+settings_file = openmc.Settings()
+settings_file.batches = batches
+settings_file.inactive = inactive
+settings_file.particles = particles
+
+# Create an initial uniform spatial source distribution over fissionable zones
+bounds = [-0.62992, -0.62992, -1, 0.62992, 0.62992, 1]
+uniform_dist = openmc.stats.Box(bounds[:3], bounds[3:], only_fissionable=True)
+settings_file.source = openmc.source.Source(space=uniform_dist)
+
+entropy_mesh = openmc.Mesh()
+entropy_mesh.lower_left = [-0.39218, -0.39218, -1.e50]
+entropy_mesh.upper_right = [0.39218, 0.39218, 1.e50]
+entropy_mesh.dimension = [10, 10, 1]
+settings_file.entropy_mesh = entropy_mesh
+settings_file.export_to_xml()
+
+###############################################################################
+#                   Initialize and run depletion calculation
+###############################################################################
+
+op = openmc.deplete.Operator(geometry, settings_file, chain_file)
+op.round_number = True
+
+# Perform simulation using the predictor algorithm
+openmc.deplete.integrator.predictor(op, time_steps, power)
+
+###############################################################################
+#                    Read depletion calculation results
+###############################################################################
+
+# Open results file
+results = openmc.deplete.ResultsList("depletion_results.h5")
+
+# Dictionnary of materials and burned nuclides
+mat_id_to_ind = results[0].mat_to_ind
+nuc_to_ind = results[0].nuc_to_ind
+
+# Obtain K_eff as a function of time
+time, keff = results.get_eigenvalue()
+                                                                                                                                                                                                                                                                                                                                                                                                                                                        
+# Obtain U235 concentration as a function of time
+time, n_U235 = results.get_atoms(mat_id_to_ind['1'], nuc_to_ind['U235'])
```

Comment:
The following would be equivalent (and much simpler)
```Python
time, n_U235 = results.get_atoms('1', 'U235')
```
You can get rid of the lines above setting `mat_id_to_ind` and `nuc_to_ind`

Reply:
Ok I had thought of showing the existence of these two dictionaries in the example but you're right they're not needed.